doc/man7/provider-asym_cipher.pod

```diff
@@ -183,7 +183,7 @@ algorithms:
 
 The type of padding to be used. The interpretation of this value will depend
 on the algorithm in use. The default provider understands these RSA padding
-modes: 1 (RSA_PKCS1_PADDING), 2 (RSA_SSLV23_PADDING), 3 (RSA_NO_PADDING),
+modes: 1 (RSA_PKCS1_PADDING), 3 (RSA_NO_PADDING),
```

Comment:
Should we renumber?

Reply:
I suggest not renumbering, since an old binary with a new shared library will fail with "unknown padding" rather than silently failing to do the expected thing.

Reply:
Since this targets 3.0 only, there can be no old binaries for this shared library yet. But keeping them is fine for me.

Reply:
There are plenty of numbers so I do not see a real reason for renumbering.